src/test/ui/consts/const_limit/const_eval_limit_reached.rs

```diff
@@ -0,0 +1,16 @@
+// check-pass
+#![feature(const_eval_limit)]
+#![const_eval_limit="2"]
+
+const CONSTANT: usize = limit();
+//~^ WARNING Constant evaluating a complex constant, this might take some time
+
+fn main() {
+    assert_eq!(CONSTANT, 1764);
+}
+
+const fn limit() -> usize {
```

Comment:
It looks like on 32 bit two steps are enough to actually get into this function instead of bailing out at the call site. Maybe just limit the test to x86_64?